x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/create_security_rule_type_wrapper.ts

```diff
@@ -118,7 +118,7 @@ export const createSecurityRuleTypeWrapper: CreateSecurityRuleTypeWrapper =
           let wroteWarningStatus = false;
 
           await ruleExecutionLogger.logStatusChange({
-            newStatus: RuleExecutionStatus['going to run'],
+            newStatus: RuleExecutionStatus.running,
```

Comment:
I'm wondering if it's even necessary to write a `running` `status change` event from security (to `event-log` at least). By and large the `execute-start` event is going to represent the same and even contain additional data (`schedule datetime`, and so `schedule delay` as well). We would still need to write to the executionSO for latest status on the Rules/Monitoring tables. 

Is there some case where `going to run` captures a unique series of events that `execute-start` can't be used for?


<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/152065797-bac0d238-d9e8-4bce-92d0-fed19c923658.png" />
</p>